src/google-maps.ts

```diff
@@ -552,4 +552,8 @@ export class GoogleMaps {
     error() {
         console.error.apply(console, arguments);
     }
+
+    resize() {
```

Comment:
This is the real "meat" of the PR. Simply adds a `resize` method which triggers the resize event on the underlying map control.